### DISCUSSION TITLE
pnpm list not displaying transitive dependencies

### DISCUSSION THREAD

**Discussion by crowleySynopsys** [pnpm/pnpm#3888] [Q&A] [answered]
I am learning about pnpm, and I'm having trouble understanding the behavior of pnpm list (ls). In the pnpm docs (https://pnpm.io/cli/list), it seems to clearly indicate that the output of the command details the direct dependencies of the project, as well as the dependencies of those dependencies (transitives), and so on.

However, when I run the command on a project only direct dependencies-as declared in the package.json-are listed. Example:

Project source: https://github.com/truenas/documentation

I clone the repo, cd into project root, run pnpm install. I then run pnpm list and get the following output:
`
dependencies:
   material-design-icons 3.0.1

devDependencies:
   autoprefixer 9.8.6
   postcss-cli 7.1.2
`
   
The output reflects the direct dependencies as declared in the project's package.json. Looking in the pnpm-lock.yaml file, though, one can see that there are dozens of other packages installed as dependencies for the declared dependencies.

I would expect those transitive dependencies to also be listed in the output of pnpm list.

Does anybody have insight on this?

**Comment by zkochan** [comment, marked as answer]
Run `pnpm ls --depth=Infinity`

**Reply by crowleySynopsys** [reply]
Thanks @zkochan .  Could have swore I tried that...but that solves it!
